### PR TITLE
Ams v2

### PR DESCRIPTION
**Pull request by leath-dub**
AMS deployment.

### CODE REVIEW

File: jobs/nginx/anime-manga-soc.hcl [hunk 1]
```diff
@@ -8,11 +8,11 @@ job "nginx-ams" {
 
     network {
       port "http" {
-        to = "80"
+        to = 3000
       }
 
       port "https" {
```
Comment (DistroByte) [inline jobs/nginx/anime-manga-soc.hcl:14]: Think this block isn't needed

File: jobs/nginx/anime-manga-soc.hcl [hunk 2]
```diff
@@ -29,17 +29,22 @@ job "nginx-ams" {
...
       ]
     }
 
     task "webserver" {
       driver = "docker"
 
       config {
-        image = "ghcr.io/dcuams/amikon-website"
+        image = "ghcr.io/dcuams/amikon-site-v2:latest"
         ports = ["http", "https"]
```
Comment (DistroByte) [inline jobs/nginx/anime-manga-soc.hcl:41]: And only "http" is needed here, but still as an array

Comment (leath-dub) [reply jobs/nginx/anime-manga-soc.hcl]: made those changes, just going to test them

Comment (DistroByte) [reply jobs/nginx/anime-manga-soc.hcl]: I believe the brackets still need to be there, [see here](https://github.com/DistroByte/nomad/blob/8996eb4c0317001ae8d684dcdbb7424c80cb7abf/jobs/web/website.hcl#L36)

**Review by DistroByte — APPROVED** [review]
lgtm now

**Comment by leath-dub** [comment]
should I remove the branch or keep it for future ref ?

**Comment by DistroByte** [comment]
The branch will always be findable, git will retain it in its history. When you "delete" a branch, you just tell git to no longer track it as active. I suggest you set up this repo to auto delete merged branches and maybe require approval to merge PRs

**Pull request merged**